lib/wx/c_src/gen/wxe_wrapper_5.cpp

```diff
@@ -2162,7 +2162,7 @@ void wxMenuItem_GetBitmap(WxeApp *app, wxeMemEnv *memenv, wxeCommand& Ecmd)
   wxMenuItem *This;
   This = (wxMenuItem *) memenv->getPtr(env, argv[0], "This");
   if(!This) throw wxe_badarg("This");
-  const wxBitmap * Result = &This->GetBitmap();
+  wxBitmap * Result = new wxBitmap(This->GetBitmap()); app->newPtr((void *) Result,3, memenv);;
```

Comment:
Nit: the generation can add a duplicated semicolon at the end, can it be avoided? Also, this code would be easier to read if it was split into 2 separate lines.

Reply:
I have not noticed the ";;", nice catch but I will not fix it in this patch.
The one-liner is intentional because I thought it was easier to read that all work was done.

Reply:
👍